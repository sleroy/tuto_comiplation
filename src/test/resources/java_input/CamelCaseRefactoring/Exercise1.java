
public class Exercise1 {


	// CamelCaseMethodName refactoring will rename this method
	public void not_camel_case_method_similar_to_local_() {

	}
	public void not_camel_case_method_similar_to_local1() {

	}

	// CamelCaseMethodName refactoring will rename this method
	public void alreadyCamelCaseMethod() {

	}
	public void alreadyCamelCaseMethod1() {

	}

	public static void main(String[] args) {
		not_camel_case_method_similar_to_local_();
		not_camel_case_method_similar_to_local1();
		alreadyCamelCaseMethod1();

		// the method invocation refactoring should rename those 2 occurences
		this.not_camel_case_method_similar_to_local_();
		this.not_camel_case_method_similar_to_local1();
		this.alreadyCamelCaseMethod();

		ExternalObject eo = new ExternalObject();

		// the method invocation refactoring should not rename those 2 occurences
		eo.not_camel_case_method();
		eo.not_camel_case_method_similar_to_local();

	}


}
