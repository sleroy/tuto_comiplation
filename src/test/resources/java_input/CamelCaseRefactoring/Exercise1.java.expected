public class Exercise1 {
  public void notCamelCaseMethodSimilarToLocal(){
  }
  public void alreadyCamelCaseMethod(){
  }
  public static void main(  String[] args){
    ExternalObject eo=new ExternalObject();
    eo.not_camel_case_method();
    eo.not_camel_case_method_similar_to_local();
    this.notCamelCaseMethodSimilarToLocal();
    this.alreadyCamelCaseMethod();
  }
}
